src/components/Hero/Hero.js

```diff
@@ -11,4 +12,19 @@ const Hero = ({info}) => {
     )
 }
 
-export default Hero;
\ No newline at end of file
+export default Hero;
+
+Hero.propTypes = {
+    info: PropTypes.shape({
```

Comment:
This is great (PropTypes.shape). I like how we can use it to look inside the object and specify each property's value type as well